src/routes/docs/configuring/date-formats.md

```diff
@@ -1,11 +1,15 @@
 # Date Formats
 
-Files comes with two different options to let you customize the date format.
+Files comes with three different options to let you customize the date format.
 
 **Application**
 
 The default date format uses the following format _Wednesday, June 3, 2020_.
 
 **System**
 
-System is a little more compact and uses this format _6/3/2020_.
\ No newline at end of file
+System is a little more compact and uses this format _6/3/2020_.
+
+**Universal**
+
+Universal uses the defacto official international format _2021-12-31_.
```

Comment:
Can you use the same date as the other examples?

Reply:
Sure, no worries, I will change that. 

I only chose a different date to avoid confusion about the order in which the month and day appear. 

Because _2020-06-03_ could be read as _June 3, 2020_, or _March 6, 2020_. But you make a valid point, if all three options represent the same date, one could easily use inference to understand the order.

Reply:
In that case, feel free to use your original example and update the other formats instead.